### PR TITLE
Finalized and Refactored Email Verification Logic

### PR DESCRIPTION
**Pull request by songrenzhao**
### Overview of PR
- [x] Refactored SendGrip payload
- [x] Removed ORM logging from `sequelize`
- [x] Added `signUpMember.js` under `/src/services`
  - [x] Added `sendVerificationEmail` to process verification
- [x] Added `healthcheck` route to check service status
- [x] Added `createVerificationCode` to generate verification token

### Technical Changes
- Refactored `sendEmail` payload
- Added `createVerificationCode`, `prepareSendgridPayload`, `sendVerificationEmail` under `/src/services/signUpService`
- Added and refactored test cases

### CODE REVIEW

File: src/app/email/sendEmail.js [hunk 1]
```diff
@@ -2,21 +2,28 @@ import httpStatus from 'http-status';
...
 export default async function sendVerificationEmail(req, res) {
   try {
+    const { body } = req;
+    const { email, token } = body;
     sgMail.setApiKey(config.sendGridApiKey);
-    const data = {
-      from: 'no-reply@example.com',
-      to: req.body.email,
-      subject: 'UniFree - Verify Your Email',
-      html: `<img width=200px
```
Comment (songrenzhao) [inline src/app/email/sendEmail.js]: Not needed, we will just send text. Thank you for your effort

File: src/app/email/sendEmail.js [hunk 2]
```diff
@@ -2,21 +2,28 @@ import httpStatus from 'http-status';
...
+    subject: 'UniFree - Verify Your Email',
+    text: `Your security token is ${token}`,
+  };
+}
+
 export default async function sendVerificationEmail(req, res) {
   try {
+    const { body } = req;
+    const { email, token } = body;
     sgMail.setApiKey(config.sendGridApiKey);
```
Comment (JiejayLan) [inline src/app/email/sendEmail.js]: Does this mean we need to set the API key every time we need to send out an email?

File: src/app/email/sendEmail.js [hunk 3]
```diff
@@ -2,21 +2,28 @@ import httpStatus from 'http-status';
...
-            src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRDRhkao
-                 D8hbQIlFIjbWW3hsAOqwryuDefYqCEXjzsS4Yr-wHVM"
-            <br>
-            <h1>Thank you for sign up UniFree, Please enter the security code within 1 minute</h1><br/>
-            <h3>Your security code is: ${req.body.token} </h3>`,
+    const data = prepareSendgridPayload(email, token);
+    const [sendGridResponse] = await sgMail.send(data);
+    const { headers } = sendGridResponse;
+    const { date } = headers;
+    const response = {
```
Comment (JiejayLan) [inline src/app/email/sendEmail.js]: It looks good. But I want to ask you some questions tonight

File: src/app/email/sendEmail.js (hunk 2 above)
Comment (songrenzhao) [reply src/app/email/sendEmail.js]: We shouldn't, will be discussed in class.

File: src/app/verify/__tests__/index.js [hunk 4]
```diff
@@ -18,27 +18,23 @@ describe('App test suite', () => {
   describe('sendEmail test suite', () => {
     const payload = {
       email: 'example@ccny.edu',
-      token: '123',
     };
-    let undefinedvar;
     it('Should send out an email', async () => {
       const response = await request(app)
-        .post('/verify/email')
+        .post('/api/v1/verify')
```
Comment (JiejayLan) [inline src/app/verify/__tests__/index.js]: can we explain what "v1" here stand for?

**Pull request merged**